src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -242,6 +242,30 @@ public void MaxRequestContentBufferSize_SetInvalidValue_ThrowsArgumentOutOfRange
             }
         }
 
+        [OuterLoop]
```

Comment:
Since the test is not skipped/disabled on Unix, we should run Outerloop test for Unix as well.

Reply:
And add UWP and NETFX as well. In general, adding new networking tests requires those in addition to Windows.  And you probably don't need both x86 and x64 Windows. One is probably fine for outerloop.

Reply:
I took a look at the test run [here](https://mc.dot.net/#/user/rmkerr/pr~2Fjenkins~2Fdotnet~2Fcorefx~2Fmaster~2F/test~2Ffunctional~2Fcli~2F/55ed935565acd1fb84980763723c78c6dfb827cd/workItem/System.Net.Http.Functional.Tests) that ran before I moved the test to Outerloop. It looks like the issue is also present in CurlHandler, since the test is failing on Linux. I'll go ahead and take a look into the handler to see if there is an obvious solution.

Reply:
Thanks

Reply:
I think that's the right spot. It looks like at the end of `SetPossibleRedirectForLocationHeader` we call `SetRequestHeaders`, which adds all of the request headers including the Authorization header. I've got a rough fix and am getting set up to test it now.